### PR TITLE
Free Call in `NodeJS` SDK. 

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: packages/core/src/MPEContract.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class MPEContract {
    * @type {string}
    */
   get address() {
-    return this.contract.address;
+    return this._contract.options.address;
```
Comment (ksridharbabuus) [inline packages/core/src/MPEContract.js:35]: What is need to have separately for options?
I like the previous approach of contract.address.

File: packages/core/src/Account.js [hunk 2]
```diff
@@ -184,7 +153,7 @@ class Account {
   }
 
   async _transactionCount() {
-    const address = await this.getAddress()
+    const address = await this.getAddress();
```
[the remark this answers is not in the document]
Comment (Vivek205) [reply packages/core/src/Account.js:156]: Yeah, we use the async call to get the accounts from eth (`eth_requestAccounts`).
The sync call has been deprecated. 
This change was made as part of the web3 migration

File: design-docs/class-diagram.plantuml [hunk 3]
```diff
@@ -61,6 +68,7 @@ MetadataProvider <|-- IPFSMetadataProvider
 
 class MPEContract {
   + address: string
+  + networkId: Number
```
[the remark this answers is not in the document]
Comment (Vivek205) [reply design-docs/class-diagram.plantuml:71]: Yeah we have the network in the SDK config, it is being passed to the MPE Contract class in the constructor

File: packages/core/src/MPEContract.js (hunk 1 above)
Comment (Vivek205) [reply packages/core/src/MPEContract.js:35]: the `contract.address` is being undefined at times. 
The new property `contract.options.address` is being more stable. 
It is part of the latest web3 https://web3js.readthedocs.io/en/v1.3.0/web3-eth-contract.html#options-address

**Review by ksridharbabuus — APPROVED** [review]
Approved. Need to do end to end test once web sdk.

**Pull request merged**